glang.spec

```diff
@@ -0,0 +1,61 @@
+Name:           glang
+Version:        0.1.3
```

Comment:
Where does this number come from :upside_down_face:?

Reply:
I started at `0.1.0` and incremented the patch every time I wanted to push to `pypi` :)